src/main/java/frc/robot/drivetrain/ImuIO.java

```diff
@@ -0,0 +1,21 @@
+package frc.robot.drivetrain;
+
+import edu.wpi.first.math.geometry.Rotation2d;
+import org.littletonrobotics.junction.AutoLog;
+
+public interface ImuIO {
```

Comment:
It is unclear to me how this class is used. Can you add a comment or provide a link?

Reply:
The akit docs describe how IO layers work

https://github.com/Mechanical-Advantage/AdvantageKit/blob/main/docs/DATA-FLOW.md
https://github.com/Mechanical-Advantage/AdvantageKit/blob/main/docs/RECORDING-INPUTS.md